crash on resize

```
panic: runtime error: slice bounds out of range          

goroutine 20 [running]:   
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.(*Edit).KeyHandler(0xc42018e240, 0x2000200000, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0xc)         
        /home/pedro/go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/edit.go:199 +0x34d                        
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.(*Window).keyHandler(0xc42016c3c0, 0x200000, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0xc420064e60, ...)                                                 
        /home/pedro/go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/window.go:237 +0xb8                       
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.initKeyHandler.func1()                     
        /home/pedro/go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/ttk.go:320 +0x11e                         
github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.init.4.func1(0xc420072390, 0xc42009a700, 0xc4200722b8, 0xc42008e1a0) 
        /home/pedro/go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/ttk.go:283 +0xfd
created by github.com/companyzero/zkc/vendor/github.com/companyzero/ttk.init.4                          
        /home/pedro/go/src/github.com/companyzero/zkc/vendor/github.com/companyzero/ttk/ttk.go:286 +0x22a        
```

How to reproduce:

1. Start zkclient on a standard 80x24 terminal;
2. Type "the quick brown fox jumps over the lazy dog the quick brown fox jumps over the lazy dog the quick";
3. Type space twice;
4. Resize the terminal from 80 to 104 columns;
5. Press any key to crash zkc.

--- a/edit.go
+++ b/edit.go
@@ -291,6 +291,7 @@ func (e *Edit) Resize() {
 			// end of text
 			if len(e.display) < e.trueW-1 {
 				e.cx = e.trueX + len(e.display)
+				e.at = 0
 			} else {
 				e.cx = e.trueX + e.trueW - 1
 				e.at = len(e.display) - e.trueW + 1

